ReSharper.FSharp/src/FSharp/FSharp.Psi.Intentions/src/Intentions/FunctionAnnotationAction.fs

```diff
@@ -238,20 +220,21 @@ type FunctionAnnotationAction(dataProvider: FSharpContextActionDataProvider) =
         let binding = dataProvider.GetSelectedElement<IBinding>()
         if isNull binding then false else
         if not (isAtBindingKeywordOrReferencePattern dataProvider binding) then false else
-        SpecifyTypes.getAvailability binding |> _.IsAvailable
+        SpecifyTypes.isAvailable binding |> _.IsAvailable
```

Comment:
This looks strange now. `isAvailable` seems to imply `bool` return type.

Reply:
```fsharp
SpecifyTypes.getAvailableOptions binding |> _.HasOptions
SpecifyTypes.getAvailableOptions decl |> _.CanSpecifyReturnType
````